R/RcppExports.R

```diff
@@ -2,26 +2,26 @@
 # Generator token: 10BE3573-1514-4C36-9D1C-5A225CD40393
 
 blockpfGaussianOpt_impl <- function(data, part, lag) {
-    .Call(`_RcppSMC_blockpfGaussianOpt_impl`, data, part, lag)
+    .Call('RcppSMC_blockpfGaussianOpt_impl', PACKAGE = 'RcppSMC', data, part, lag)
```

Comment:
You are using an Rcpp version older than 0.12.12.  

Please upgrade, and re-create the file, or explain why you must use on old version, possibly over at the Rcpp repo with an issue ticket there.